File: Lista1/Python/ex_10.py
cont_primo = 0
numero = int(input('Digite um número para saber se é primo ou não: '))
for c in range(1, numero+1):
    if numero % c == 0:
        cont_primo += 1
if cont_primo == 2 or numero == 1:
    print(f'O número {numero} é primo')
else:
    print(f'O número {numero} não é primo')

File: Lista1/Java/ex_10.java
import java.util.Scanner;
public class ex_10
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
        int contador = 0;
        System.out.println("Digite um número para saber se ele é um número primo: ");
        int numero = sc.nextInt();
        for (int c = 1; c <= numero; c++) {
            if (numero % c == 0) {
                contador ++;
            }
        }
        if (contador == 2 || numero == 1) {
        System.out.println("O número " + numero + " é primo!");
        }
        else {System.out.println("O número " + numero + " não é primo!");
       }        
    }
}